UnityProject/Assets/Scripts/HealthV2/Living/Damage/BodyPartTraumaDamage.cs

```diff
@@ -342,6 +368,37 @@ private void TakeBurnDamage(float burnDamage)
 			}
 		}
 
+
+		/// <summary>
+		/// Blunt damage is meant to reduce the bone/mechanical organs's health.
+		/// A less healthy bone is more likely to break and cause issues.
+		/// This damage should ONLY be applied to bodyParts that have CanBeBroken ticked.
+		/// </summary>
+		private void TakeBluntDamage(float bluntDamage)
+		{
+			if(CanBeBroken){health -= MultiplyTraumaDamage(bluntDamage);} //Incase we're damaging a mechanical body part.
+			foreach(ItemSlot organ in OrganStorage.GetItemSlots())
+			{
+				BodyPart bone = organ.ItemObject.GetComponent<BodyPart>();
+				if(bone != null)
+				{
+					if(bone.CanBeBroken)
+					{
+						if(bone.SelfArmor.Melee < bluntDamage){bone.health -= MultiplyTraumaDamage(bluntDamage);}
+						bone.CheckBluntDamageLevels();
+					}
+				}
+			}
+		}
+
+		private void CheckBluntDamageLevels()
+		{
+			if(health >= 76){currentBluntDamageLevel = BluntDamageLevels.HEALTHY;}
+			if(health <= 75){currentBluntDamageLevel = BluntDamageLevels.MODERATE;}
+			if(health <= 45){currentBluntDamageLevel = BluntDamageLevels.SEVERE;}
+			if(health <= 25){currentBluntDamageLevel = BluntDamageLevels.CRITICAL;}
```

Comment:
not a big deal but using else's is a cleaner way to do this

Reply:
There is also ``float.IsBetween(min, max)``

Reply:
On it